specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5773,13 +5773,238 @@
+        "allMetadata",
+        "contentAndMetadata"
+      ],
+      "x-ms-enum": {
+        "name": "BlobIndexerDataToExtract",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "storageMetadata",
+            "name": "StorageMetadata",
+            "description": "Indexes just the standard blob properties and user-specified metadata."
+          },
+          {
+            "value": "allMetadata",
+            "name": "AllMetadata",
+            "description": "Extracts metadata provided by the Azure blob storage subsystem and the content-type specific metadata (for example, metadata unique to just .png files are indexed)."
+          },
+          {
+            "value": "contentAndMetadata",
+            "name": "ContentAndMetadata",
+            "description": "Extracts all metadata and textual content from each blob."
+          }
+        ]
+      },
+      "x-nullable": true,
+      "default": "contentAndMetadata",
+      "description": "Specifies the data to extract from Azure blob storage and tells the indexer which data to extract from image content when \"imageAction\" is set to a value other than \"none\".  This applies to embedded image content in a .PDF or other application, or image files such as .jpg and .png, in Azure blobs."
+    },
+    "ImageAction": {
+      "type": "string",
+      "enum": [
+        "none",
+        "generateNormalizedImages",
+        "generateNormalizedImagePerPage"
+      ],
+      "x-ms-enum": {
+        "name": "BlobIndexerImageAction",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "none",
+            "name": "None",
+            "description": "Ignores embedded images or image files in the data set.  This is the default."
+          },
+          {
+            "value": "generateNormalizedImages",
+            "name": "GenerateNormalizedImages",
+            "description": "Extracts text from images (for example, the word \"STOP\" from a traffic stop sign), and embeds it into the content field.  This action requires that \"dataToExtract\" is set to \"contentAndMetadata\".  A normalized image refers to additional processing resulting in uniform image output, sized and rotated to promote consistent rendering when you include images in visual search results. This information is generated for each image when you use this option."
+          },
+          {
+            "value": "generateNormalizedImagePerPage",
+            "name": "GenerateNormalizedImagePerPage",
+            "description": "Extracts text from images (for example, the word \"STOP\" from a traffic stop sign), and embeds it into the content field, but treats PDF files differently in that each page will be rendered as an image and normalized accordingly, instead of extracting embedded images.  Non-PDF file types will be treated the same as if \"generateNormalizedImages\" was set."
+          }
+        ]
+      },
+      "x-nullable": true,
```

Comment:
What does it mean for a type to be nullable? I thought this attribute only applied to properties, not types.

Reply:
Removed in commit https://github.com/Azure/azure-rest-api-specs/pull/9900/commits/bb01e6970084e07ec0e4f63df489d9e86ae62882